pybind11/CMakeLists.txt

```diff
@@ -0,0 +1,79 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(pybind11)
+
+find_package(catkin REQUIRED)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+
+include(pybind)
+include(GNUInstallDirs)
+include(CMakePackageConfigHelpers)
+
+file(MAKE_DIRECTORY include)
+catkin_package(
+  INCLUDE_DIRS include pybind11/include ${EIGEN3_INCLUDE_DIR}
```

Comment:
It gets called inside one of the cmake files included from pybind11 which also gets called in dependant packages via ```pybind.cmake```, so I'm guessing not.